packages/services/src/services/Translation/Translation.js

```diff
@@ -24,7 +24,11 @@ const _host =
  * @type {string}
  * @private
  */
-const _endpoint = `${_host}/common/v18/js/data/jsononly`;
+const _ddsEndpointDefault =
+  (process &&
+    (process.env.REACT_APP_DDS_TRANSLATION_ENDPOINT ||
+      process.env.DDS_TRANSLATION_ENDPOINT)) ||
+  '/common/v18/js/data/jsononly';
```

Comment:
One way to approach the sessionKey logic is to probably have two variables:

```
const _ddsEndpointDefault = '/common/carbon-for-ibm-dotcom/translations/masthead-footer';

const _ddsEndpoint = (process &&
    (process.env.REACT_APP_DDS_TRANSLATION_ENDPOINT ||
      process.env.DDS_TRANSLATION_ENDPOINT)) || _ddsEndpointDefault;
```

Then:

```
if(_ddsEndpointDefault !== _ddsEndpoint) {
   ...do logic to set a different session key...
}
```

Reply:
done!